Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,40 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            string firstName;
+            string lastName;
+
+            int age;
+            double weight;
+            double height;
+
+            double bmi;
+            for (int i = 0; i < 2; i++)
+            {
+                Console.WriteLine("What is your First Name?");
+                firstName = Console.ReadLine();
+
+                Console.WriteLine("\nWhat is your Last Name?");
+                lastName = Console.ReadLine();
+
+                Console.WriteLine("\nWhate is your age?");
+                age = Convert.ToInt32(Console.ReadLine());
+
+                Console.WriteLine("\nWhate is your weight(kg)?");
+                weight = Convert.ToDouble(Console.ReadLine());
+
+                Console.WriteLine("\nWhate is your height(cm)?");
+                height = Convert.ToDouble(Console.ReadLine());
+
+                Console.WriteLine("\n" + firstName + " " + lastName + " is a " + age + " year old, their weight is " +
+                    weight + " kg and their height is " + height + " cm.");
```

Comment:
While this is perfectly fine its worth having a look at string interpolation.
```cs
string userName = "<Type your name here>";
string date = DateTime.Today.ToShortDateString();

// Use string interpolation to concatenate strings.
string str = $"Hello {userName}. Today is {date}.";
System.Console.WriteLine(str);

str = $"{str} How are you today?";
System.Console.WriteLine(str);
```
Source: https://docs.microsoft.com/en-us/dotnet/csharp/how-to/concatenate-multiple-strings#code-try-2

Reply:
Oh thank you, I have never heard of this before.